.github/workflows/swiftlint.yml

```diff
@@ -0,0 +1,42 @@
+name: SwiftLint
+
+on:
+  push:
+    branches: stable
+    paths:
+      - '.github/workflows/swiftlint.yml'
+      - '.swiftlint.yml'
+      - '**/*.swift'
+  pull_request:
+    paths:
+      - '.github/workflows/swiftlint.yml'
+      - '.swiftlint.yml'
+      - '**/*.swift'
```

Comment:
```suggestion
    paths: &trigger_paths
      - '.github/workflows/swiftlint.yml'
      - '.swiftlint.yml'
      - '**/*.swift'
  pull_request:
    paths: *trigger_paths
```

Reply:
Github does not support this yaml thing **yet** (aliases/replacement/whatever)

Reply:
Dammit